### ISSUE TITLE
Why don't you add the URL of the Tutorial to the new warning message?

### ISSUE THREAD

**Issue by ShotaAk** [colcon/colcon-core#467]
A new warning message from colcon (and this issue https://github.com/colcon/colcon-core/issues/465 ) made me realize that I was using the workspace in the wrong way.

```sh
WARNING:colcon.colcon_core.verb:Some selected packages are already built in one or more underlay workspaces:
```

If the warning message had a link to the following ROS2 Tutorials, I would have noticed the problem earlier.

http://docs.ros.org.ros.informatik.uni-freiburg.de/en/foxy/Tutorials/Workspace/Creating-A-Workspace.html#source-the-overlay

Thank you.

**Comment by cottsay** [comment]
I don't believe that this warning is displayed under these circumstances after it was recently changed. If I understand correctly, the warning actually targets other "unsafe" circumstances and not the one discussed in the documentation you linked.

Please update to the latest `colcon` packages and try to reproduce the warning. If you're still seeing it, we should open an issue on the `colcon-override-check` repository.
